avocado/core/nrunner.py

```diff
@@ -279,6 +279,26 @@ class BaseRunner:
     def __init__(self, runnable):
         self.runnable = runnable
 
+    def prepare_status(self, status_type, additional_info=None):
+        """Prepare a status dict with some basic information.
+
+        This will add the keywork 'status' and 'time' to all status.
+
+        :param: status_type: The type of event ('started', 'running',
+                             'finished')
+        :param: addional_info: Any addional information that you
+        would like to
+                               add to the dict. This must be a dict.
+
+        :rtype: dict
+        """
+        status = {'status': status_type,
+                  'time': time.time()}
```

Comment:
The `tap` runner currently reports a `timestamp` that is identical to this. We need to either use one or the other.

Reply:
@clebergnu that is right. my fault, I was not paying attention to the other runners. I will change it during v2.